python/Makefile

```diff
@@ -10,6 +10,16 @@ integration_tests:
 	@echo integration tests...
 	nosetests --rednose ./tests/scripts
 
+e2e_setup:
+	@echo end to end tests set up...
+	../common/scripts/e2e_tests_prerequisites
+
+e2e_tests:
```

Comment:
```suggestion
e2e_tests: e2e_setup
```

Reply:
e2e_setup pulls refreshable access tokens that can be used for many iterations of e2e_tests. e2e_setup requires manually hitting the approve button on the OAuth page, so it's best to do it as infrequently as possible.

Reply:
done. see above.